### ISSUE TITLE
Slicing bug - Center of Buildplate

### ISSUE THREAD

**Issue by Zillaccia** [Ultimaker/Cura#13643]
### Application Version

5.2.1

### Platform

windows 10

### Printer

FLSUN QQ_S pro

### Reproduction steps

Using the last parametrer used, i launced the slice on the design already suicefully sliced with the v5.2.0, incurring in a bug

### Actual results

a not sliced file

### Expected results

[isabug.zip](https://github.com/Ultimaker/Cura/files/9854503/isabug.zip)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

hi

**Comment by GregValiant** [comment]
Thanks for the report.
I can duplicate the problem with your project file.  A workaround is to move and/or rotate the model on the build plate.
Someone from the Cura team will take a look at this.

**Comment by GregValiant** [comment]
#13685 is a symmetrical object that won't slice unless rotated 90° and it really shouldn't matter on that one.

**Comment by MariMakes** [comment]
Hey @Zillaccia,

Welcome to the Ultimaker Cura Github 🚀 
Sorry, it took us a while to get back to you 😞 

Thanks for your report 👍
I can recreate your issue on my side:
![image](https://user-images.githubusercontent.com/40423138/205098609-e76918d8-2df9-4ef4-b1a7-f77a279c15b9.png)

I've noticed that if you move the model slightly from the center of the buildplate, you are able to slice it. 
I hope that helps you get unstuck. 
![image](https://user-images.githubusercontent.com/40423138/205099366-ad399c8b-3377-4820-b79e-75ffdb400031.png)


I've also started collecting issues that have the same "workaround" and I'll bring it up to the team to see if it can be resolved.
Fingers crossed that they can find something 🤞 

Thanks again! 💪
